src/config_handling.py

```diff
@@ -53,14 +56,18 @@ def init_config(app):
         app.config[config] = value
 
 
+def is_dev_mode() -> bool:
+    value = os.environ.get(MEDIA_DEV_MODE_ENVIRONMENT_VAR, None)
+    return value is not None and value.lower() in ("1", "on", "yes", "true")
+
+
 def get_config_from(app, config):
-    path = os.environ.get(config)
-    if not path:
-        app.logger.critical(f"Did not find environment variable for '{config}'")
-        sys.exit(1)
-    try:
+    path = os.environ.get(config + "_FILE", None)
+    if is_dev_mode():
+        value = DEV_SECRET
+    elif path is not None:
         with open(path) as file_:
-            return file_.read()
-    except Exception:  # noqa
-        app.logger.critical(f"Reading path '{path}' from config '{config}' went wrong.")
-        sys.exit(1)
+            value = file_.read()
+    else:
+        value = os.environ.get(config, None)
```

Comment:
I think, here you can write

```suggestion
        value = os.environ.get(config, get_default_for(config))
```

Reply:
Done.